### PR TITLE
docs(content/en/docs/rules): add warning for `Via Custom Rule Definion` section

### PR DESCRIPTION
**Pull request by leogr**
See also: https://github.com/falcosecurity/falco/issues/1537

Signed-off-by: Leonardo Grasso <me@leonardograsso.com>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind user-interface

/kind content

> /kind translation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area blog

/area documentation

> /area videos

**What this PR does / why we need it**:
See https://github.com/falcosecurity/falco/issues/1537

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

/cc @Rajakavitha1

### CODE REVIEW

File: content/en/docs/rules/_index.md [hunk 1]
```diff
@@ -251,11 +251,20 @@ Last but not least, it's also possible to just disable a by default enabled rule
   enabled: false
 ```
 
-This is especially useful for rules which do not provide a `consider_*` macro in it's default condition. Again, ensure the falco configuration files are loaded in the right order.
+This is especially useful for rules which do not provide a `consider_*` macro in it's default condition. Again, ensure the Falco configuration files are loaded in the right order.
```
Comment (Rajakavitha1) [inline content/en/docs/rules/_index.md:254]: How about?
Last but not the least, you can just disable a rule that is enabled by default using a combination of the `append: true` and `enabled: false` rule properties.
This is especially useful for rules which do not provide a `consider_*` macro in the default condition.

Ensure that the custom configuration file loads after the default configuration file. You can configure the right order using multiple `-r` parameters, directly inside the falco configuration file `falco.yaml` through `rules_file`. If you are using the official Helm chart, then configure the order with the `falco.rulesFile` value.

For example to disable the `User mgmt binaries` default rule in `/etc/falco/falco_rules.yaml` define a custom rule in `/etc/falco/rules.d/custom-rules.yaml`:

```yaml
- rule: User mgmt binaries
  append: true
  enabled: false
```


 {{% pageinfo color="warning" %}}
 There appears to be a bug with this feature that we are looking into. If `enabled: false` doesn't work, you can use the following workaround as an alternative:
 ```yaml
 - rule: User mgmt binaries
   condition: and (never_true)
   append: true
 ```
 {{% /pageinfo %}}

Comment (leogr) [reply content/en/docs/rules/_index.md]: Thank you! much better

**Review by leodido — APPROVED** [review]

**Pull request merged**